### ISSUE TITLE
Info: Windows 10 insider removes yakyak executable

### ISSUE THREAD

**Issue by survivor303** [yakyak/yakyak#1132]
Windows insider remove yakyak executable file after Windows get updated.

Before that, even Kaspersky av says that the file behavior is virus like (PDM:Trojan.Win32.Generic) and moved the file to quarantine.

This executable was from prebuild rolling version, **yakyak-1.5.5-win32-x64.zip**.

If i scan that file now (22.02.2020), with Kaspersky and Windows defender, their heurestic scans dosnt detect that trojan anymore there. So somewhere between 13.02.2020 - 21.02.2020, there was something off, on these engines heurestics that give them false reason to act.

There is virustotal results for the latest yakyak.exe binary file:
https://www.virustotal.com/gui/file/955e01d69e9ad546099284f799a4ab679d618fa69439216e4d218dd9579da902/detection

Like we see on that report, only one av engine sees something odd on that file, but it cannot detect exact what kind of malware there is if none, but it detects something which can be malicious. (just like kaspersky prompt earlier).

Bottom line: There isnt any security issue on here, at least from yakyak project.

Anyways, perhaps devs can look in to this, and try resolve what triggers these av engines heurestics to think that there is something odd?
